src/routes/authentication.rs

```diff
@@ -116,7 +115,8 @@ async fn get_credential_requirements_handler(
     };
 
     // encode and encrypt it for the receiver
-    let msg_json = serde_json::to_string(&msg).unwrap();
+    let msg_json = serde_json::to_string(&msg)
+        .map_err(|_| Error::Internal("serialization error".to_string()))?;
```

Comment:
Why don't we bubble up the error as well? Might help with debugging.

Reply:
ab81af40c8b6